app/assets/javascripts/observation_accuracy_experiments/show.js.erb

```diff
@@ -0,0 +1,124 @@
+$(function() {
```

Comment:
Some syntax here doesn't conform to our JS linting rules defined at https://github.com/inaturalist/inaturalist/blob/main/app/assets/javascripts/.eslintrc. I'd recommend getting a JS linter set up in your editor. Alternatively you can run eslint directly, e.g. `./node_modules/.bin/eslint app/assets/javascripts/observation_accuracy_experiments/show.js.erb`